### PR TITLE
update changelog

### LINKED ISSUE #193: Update Changelog with progress since 0.3.12
The latest airbrake-js release (as of this writing) is v0.5.8 from December 2015. The Changelog was last updated for v0.3.12 in March 2015.

I don't need detail for every single release, but @vmihailenco could you summarize what changed over this time? Here's a GitHub compare view:

https://github.com/airbrake/airbrake-js/compare/v0.3.12...v0.5.8

### PR DESCRIPTION
**Pull request by shime**
closes #193

### CODE REVIEW

**Comment by zefer** [comment]
Thanks @shime! @jrr kindly highlighted some important changes in https://github.com/airbrake/airbrake-js/issues/193#issuecomment-197153639 - would it be possible to include these in the appropriate places in this changelog?

**Comment by shime** [comment]
@zefer it's hard to include this without @vmihailenco help

**Comment by shime** [comment]
@zefer - I've included @jrr's notes in the changelog. wording is sometimes a bit different to be more understandable.

**Comment by zefer** [comment]
Thanks @shime & @jrr. LGTM.
